db/schema.rb

```diff
@@ -56,6 +48,35 @@
     t.index ["assessment_id"], name: "index_benefit_receipts_on_assessment_id"
   end
 
+  create_table "capital_items", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+    t.uuid "capital_summary_id"
+    t.string "type", null: false
+    t.string "description", null: false
+    t.decimal "value", default: "0.0", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["capital_summary_id"], name: "index_capital_items_on_capital_summary_id"
+  end
+
+  create_table "capital_summaries", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+    t.uuid "assessment_id"
+    t.decimal "total_liquid", default: "0.0", null: false
+    t.decimal "total_non_liquid", default: "0.0", null: false
+    t.decimal "total_vehicle", default: "0.0", null: false
+    t.decimal "total_property", default: "0.0", null: false
+    t.decimal "total_mortgage_allowance", default: "0.0", null: false
```

Comment:
Do we really need all these defaults and null declarations?

Reply:
we can do without the not nulls I suppose, but if we don't have a default value, we have to set everything up or make sure we cope with nils.